.github/workflows/tests-cifuzz.yaml

```
@@ -0,0 +1,30 @@
+name: CIFuzz
+on:
+  pull_request:
+    paths-ignore:
+      - 'Documentation/**'
+permissions: read-all
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build Fuzzers
+      id: build
+      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@723bdbc7a8ee1e95af24284583b25d41efc0bd41
+      with:
+        oss-fuzz-project-name: 'cilium'
+        dry-run: false
+        language: go
+    - name: Run Fuzzers
+      uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@723bdbc7a8ee1e95af24284583b25d41efc0bd41
+      with:
+        oss-fuzz-project-name: 'cilium'
+        fuzz-seconds: 600
+        dry-run: false
+        language: go
+    - name: Upload Crash
+      uses: actions/upload-artifact@723bdbc7a8ee1e95af24284583b25d41efc0bd41
```

I guess there was a mixup because this seems to be the same commit as the fuzzing actions above, and of course it does exist over at `actions/upload-artifact` 😄

It should be this one instead, corresponding to the latest release 2.3.0: https://github.com/actions/upload-artifact/releases/tag/v2.3.0
```suggestion
      uses: actions/upload-artifact@da838ae9595ac94171fa2d4de5a2f117b3e7ac32
```

---

ah of course - done!